pipeline/transform/user_interests.py

```diff
@@ -3,16 +3,37 @@
 import prefect
 from prefect import task
 
-from pipeline.types import RawUserInterests, User
+from pipeline.types import Interest, RawUserInterests, User
 
 
 @task
 def augment_users_with_interests(
     users: List[User], raw_interests: RawUserInterests
 ) -> List[User]:
-    # TODO: Implement your transform task
 
     logger = prefect.context.get("logger")
-    logger.info("Chris")
+    logger.info(
+        "Finished issues #103 & #194 (Chris); wrote additional test cases as well"
+    )
+
+    # TODO: Implement your transform task
+
+    for user in users:
+
+        interest_lst = []
+
+        try:
+
+            for key, value in raw_interests[user.uid].items():
+
+                if value:
+
+                    interest_lst.append(Interest(code=key, name=key))
+
+                    user.interests = interest_lst
+
+        except KeyError:
+
+            break
```

Comment:
If you `break` here, then we will skip all the remaining users. You can use `continue` instead.